{
    "data": [
        {
            "test_name": "python.functions.RedfieldTest.RedfieldTest",
            "latest_failed_build": "3899",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "3901",
                    "3900",
                    "3899",
                    "3898",
                    "3897",
                    "3896",
                    "3895",
                    "3894",
                    "3893",
                    "3892",
                    "3891",
                    "3890",
                    "3889",
                    "3888",
                    "3887",
                    "3886",
                    "3885",
                    "3884",
                    "3883",
                    "3882",
                    "3881",
                    "3880",
                    "3879",
                    "3878",
                    "3877",
                    "3876",
                    "3875",
                    "3874",
                    "3873",
                    "3872"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Failed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.functions.RedfieldTest.RedfieldTest",
                "build": "3899",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "4.63",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.6.20230503.1803.dev14\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.02 seconds\nPython-[Warning] Property NIterations to Fit does not apply to any of the input workspaces\nFit-[Notice] Fit started\nFit-[Notice] Fit status: success\nFit-[Notice] Stopped after 1 iterations\nFit-[Notice] Fit successful, Duration 0.00 seconds\n..F\n======================================================================\nFAIL: test_function_output (RedfieldTest.RedfieldTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/test/python/plugins/functions/RedfieldTest.py&quot;, line 27, in test_function_output\n    self.fail(msg.format(*[str(i) for i in (output, input, expected)]))\nAssertionError: Computed output [3.62599778 0.40031891 0.10911462 0.04931991] from input [   0.          333.33333333  666.66666667 1000.        ] unequal to expected: [0.091847597, 0.076278678, 0.05056509, 0.03237545]\n\n----------------------------------------------------------------------\nRan 3 tests in 0.104s\n\nFAILED (failures=1)\n\n          Start 2047: mantidqt_qt5.test_codeeditor_tab_presenter.test_codeeditor_tab_presenter\n"
            }
        },
        {
            "test_name": "APITest_LogManagerTest",
            "latest_failed_build": "3896",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "3901",
                    "3900",
                    "3899",
                    "3898",
                    "3897",
                    "3896",
                    "3895",
                    "3894",
                    "3893",
                    "3892",
                    "3891",
                    "3890",
                    "3889",
                    "3888",
                    "3887",
                    "3886",
                    "3885",
                    "3884",
                    "3883",
                    "3882",
                    "3881",
                    "3880",
                    "3879",
                    "3878",
                    "3877",
                    "3876",
                    "3875",
                    "3874",
                    "3873",
                    "3872"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "None",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "APITest_LogManagerTest",
                "build": "3896",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "1.43",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.6.20230427.1427.dev144\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nRunning 50 tests.................................TimeSeriesProperty-[Notice] Log series has 1 entries removed due to duplicated time. \n..............TimeSeriesProperty-[Notice] Log test_get_invalid_values_filter_invalid_values has 1 entries removed due to duplicated time. \n.\nIn LogManagerTest::test_removeDataOutsideTimeROI:\n/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/API/test/LogManagerTest.h:716: Error: Expected (run_input == run_expect), found ({ 80 8C 9D 94 C1 7F 00 00 ... } != { 80 8C 9D 94 C1 7F 00 00 ... })\n.\nFailed 1 of 50 tests\nSuccess rate: 98%\n\n          Start 1995: mantidqt_qt5.test_algorithm_observer.test_algorithm_observer\n"
            }
        },
        {
            "test_name": "python.scripts.AbinsPowderCalculatorTest.AbinsPowderCalculatorTest",
            "latest_failed_build": "3892",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "3901",
                    "3900",
                    "3899",
                    "3898",
                    "3897",
                    "3896",
                    "3895",
                    "3894",
                    "3893",
                    "3892",
                    "3891",
                    "3890",
                    "3889",
                    "3888",
                    "3887",
                    "3886",
                    "3885",
                    "3884",
                    "3883",
                    "3882",
                    "3881",
                    "3880",
                    "3879",
                    "3878",
                    "3877",
                    "3876",
                    "3875",
                    "3874",
                    "3873",
                    "3872"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.scripts.AbinsPowderCalculatorTest.AbinsPowderCalculatorTest",
                "build": "3892",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "4.90",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.6.20230427.1427.dev34\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nPython-[Notice] Using Euphonic library for CASTEP file import\nPython-[Warning] k-point weights do not sum to 1. Re-normalising...\nFsPython-[Notice] Using Euphonic library for CASTEP file import\n.\n======================================================================\nFAIL: test_good_case (AbinsPowderCalculatorTest.PowderCalculatorTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/scripts/test/Abins/AbinsPowderCalculatorTest.py&quot;, line 63, in _good_case\n    assert_allclose(good_data[&quot;powder&quot;][key][i], calculated_data[key][i], atol=0.001)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/numpy/testing/_private/utils.py&quot;, line 1527, in assert_allclose\n    assert_array_compare(compare, actual, desired, err_msg=str(err_msg),\nAssertionError: \nNot equal to tolerance rtol=1e-07, atol=0.001\n\nMismatched elements: 16 / 60912 (0.0263%)\nMax absolute difference: 0.00178064\nMax relative difference: 55006.92042071\n x: array([[[[ 5.852740e-03, -2.369582e-04, -1.982987e-03],\n         [-2.369582e-04,  9.593655e-06,  8.028462e-05],\n         [-1.982987e-03,  8.028462e-05,  6.718628e-04]],...\n y: array([[[[ 5.852719e-03, -2.369274e-04, -1.982975e-03],\n         [-2.369274e-04,  9.591199e-06,  8.027400e-05],\n         [-1.982975e-03,  8.027400e-05,  6.718572e-04]],...\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/scripts/test/Abins/AbinsPowderCalculatorTest.py&quot;, line 46, in test_good_case\n    self._good_case(name=self._c6h6)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/scripts/test/Abins/AbinsPowderCalculatorTest.py&quot;, line 65, in _good_case\n    raise AssertionError(f&quot;Difference in field &#x27;{key}&#x27;&quot;) from e\nAssertionError: Difference in field &#x27;b_tensors&#x27;\n\n----------------------------------------------------------------------\nRan 3 tests in 1.738s\n\nFAILED (failures=1, skipped=1)\n\n          Start  425: AlgorithmsTest_RunCombinationHelperTest\n"
            }
        },
        {
            "test_name": "DataObjectsTest_GroupingWorkspaceTest",
            "latest_failed_build": "3891",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "3901",
                    "3900",
                    "3899",
                    "3898",
                    "3897",
                    "3896",
                    "3895",
                    "3894",
                    "3893",
                    "3892",
                    "3891",
                    "3890",
                    "3889",
                    "3888",
                    "3887",
                    "3886",
                    "3885",
                    "3884",
                    "3883",
                    "3882",
                    "3881",
                    "3880",
                    "3879",
                    "3878",
                    "3877",
                    "3876",
                    "3875",
                    "3874",
                    "3873",
                    "3872"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "DataObjectsTest_GroupingWorkspaceTest",
                "build": "3891",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "0.29",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.6.20230427.1427.dev34\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nRunning 7 tests......\nIn GroupingWorkspaceTest::testGetGroupSpetraIDs:\n/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/DataObjects/test/GroupingWorkspaceTest.h:155: Error: Expected (ws-&gt;getGroupSpetraIDs(0).size() == 98), found (0 != 98)\nFailed 1 of 7 tests\nSuccess rate: 85%\n\n          Start 1206: GeometryTest_SurfaceFactoryTest\n"
            }
        }
    ]
}